### PR TITLE
Add dependencies when running the wizard

### PR DESCRIPTION
**Pull request by meggart**
On current master, binary dependencies were not added to the build environment when running the wizard, this should be fixed by this PR. I am not very familiar with the codebase, so maybe there are better ways to do this.

### CODE REVIEW

**Comment by staticfloat** [comment]
Wow, great fix @meggart! I'm going to push some commits to this branch to refactor/fix the tests a bit, then we'll merge.  :D

**Comment by staticfloat** [comment]
Okay I got a little carried away.

File: src/auditor/compiler_abi.jl [hunk 1]
```diff
@@ -158,8 +158,8 @@ function detect_cxxstring_abi(oh::ObjectHandle, platform::Platform; io::IO = std
         if isa(e, InterruptException)
             rethrow(e)
         end
-        warn(io, "$(path(oh)) could not be scanned for cxx11 ABI!")
-        Base.display_error(io, e, [])
+        @warn("$(path(oh)) could not be scanned for cxx11 ABI!")
```
Comment (fredrikekre) [inline src/auditor/compiler_abi.jl]: You can use `exception` to attach the exception:
```suggestion
        @warn("$(path(oh)) could not be scanned for cxx11 ABI!") exception=e
```
optionally with the backtrace like this:
```suggestion
        @warn("$(path(oh)) could not be scanned for cxx11 ABI!") exception=(e, catch_backtrace())
```

Comment (staticfloat) [reply src/auditor/compiler_abi.jl]: WOW I LOVE IT.

**Pull request merged**